src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -127,25 +132,29 @@ function compute_sgs_flux!(
     # apply surface BC as SGS flux at lowest level
     lg_surf = CC.Fields.local_geometry_field(axes(ρ_f))[kf_surf]
     sgs_flux_h_tot[kf_surf] =
-        CCG.Covariant3Vector(CCG.WVector(get_ρe_tot_flux(surf)), lg_surf)
+        CCG.Covariant3Vector(surface_conditions.ρ_flux_h_tot, lg_surf)
```

Comment:
The conversion to `C3` is no longer necessary, as `surface_conditions.ρ_flux_h_tot` is already a `C3`. Same for all the other fluxes.